PLATER/services/app_trapi_1_0.py

```diff
@@ -1,80 +1,43 @@
 """FastAPI app."""
 import json
 import os
-import yaml
 from typing import Any, Dict, List
 
 from fastapi import Body, Depends, FastAPI
-from fastapi.middleware.cors import CORSMiddleware
-from fastapi.openapi.utils import get_openapi
+from fastapi.responses import JSONResponse
 
-from PLATER.services.models import (
+from PLATER.models.models_trapi_1_0 import (
     Message, ReasonerRequest, CypherRequest, SimpleSpecResponse, SimpleSpecElement,
     GraphSummaryResponse, CypherResponse, PredicatesResponse
 )
-from PLATER.services.config import config
 from PLATER.services.util.bl_helper import BLHelper
 from PLATER.services.util.graph_adapter import GraphInterface
-from PLATER.services.util.logutil import LoggingUtil
 from PLATER.services.util.overlay import Overlay
 from PLATER.services.util.question import Question
+from PLATER.services.util.api_utils import get_graph_interface, get_bl_helper, construct_open_api_schema, get_example
 
-TITLE = config.get('PLATER_TITLE', 'Plater API')
-VERSION = os.environ.get('PLATER_VERSION', '1.0.0')
-
-logger = LoggingUtil.init_logging(
-    __name__,
-    config.get('logging_level'),
-    config.get('logging_format'),
-)
-
-APP = FastAPI()
-
-
-def get_graph_interface():
-    """Get graph interface."""
-    return GraphInterface(
-        config.get('NEO4J_HOST'),
-        config.get('NEO4J_HTTP_PORT'),
-        (
-            config.get('NEO4J_USERNAME'),
-            config.get('NEO4J_PASSWORD')
-        )
-    )
 
+APP_TRAPI_1_0 = FastAPI(openapi_url='/1.0/openapi.json', docs_url='/1.0/docs')
 
-def get_bl_helper():
-    """Get Biolink helper."""
-    return BLHelper(config.get('BL_HOST', 'https://bl-lookup-sri.renci.org'))
 
 
-def get_example(operation: str):
-    """Get example for operation."""
-    with open(os.path.join(
-        os.path.dirname(__file__),
-        "..",
-        "examples",
-        f"{operation}.json",
-    )) as stream:
-        return json.load(stream)
-
 
 async def reasoner_api(
         request: ReasonerRequest = Body(
             ...,
             example=get_example("reasoner"),
         ),
         graph_interface: GraphInterface = Depends(get_graph_interface),
-) -> Message:
+):
     """Handle TRAPI request."""
     request_json = request.dict()
     question = Question(request_json["message"])
     response = await question.answer(graph_interface)
-    request_json.update({'message': response})
-    return request_json
+    request_json.update({"message": response})
+    return JSONResponse(request_json)
```

Comment:
I'm curious why this is necessary. FastAPI typically generates a JSON response by default.

Reply:
This was to allow backward compatibility, i.e 1.0 responses I was running into validation errors as the default fast api response would pass through the 1.1 validation. Will reverfy with the latest,

Reply:
Latest Transplier has some changes that require some refactoring in plater. For this PR I think we should use a commit version of transpliler in our requirements and Work on that refactor on the next set of things.